content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,289 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a model registry to manage the lifecycle of the models we get from
+our experiments. Using DVC and DVC [Studio](/doc/studio) we will set up a model
+registry where we can discover, share, deploy and audit all our models and which
+will serve as the single source of truth for our model management.
+
+## Adding models
+
+Let's now train a model and add it to the model registry. We will be using
+[DVCLive](/doc/dvclive) and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to cache the
+model with dvc and then add it to the dvc remote and the model registry once we
+call `dvc push`. Open the training script `src/train.py` in our example
+repository and you will see the following code under the `with Live(...)`
+context:
```

Comment:
```suggestion
We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to
<abbr>cache</abbr> themodel with dvc and add it to the the model registry. 
Open the training script `src/train.py` in our example
repository and you will see the following code under the `with Live(...)`
context:
```

Sorry, after reflecting on this more, let's not get into `dvc push` here since it will be a focus on the next page.

Reply:
Yeah, that makes sense, the users will only need to interact with the artifacts in that part. Here we can just keep things. That also means forking the repo is going to be easier for this chapter - You still need to fork it, sign up to Studio, install it on your GH account and add the project. But you don't need to set up a remote yet.